src/Symfony/Component/Notifier/Event/MessageEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Event;
+
+use Symfony\Component\EventDispatcher\Event;
```

Comment:
It's deprecated, the one in the Contracts should be used instead.

Reply:
nvm, updated